### PR TITLE
Revisions

### PR DESCRIPTION
**Pull request by kghopson**
Updated the first paragraph and last sentence. Made a handful of other minor changes for readability.

### CODE REVIEW

File: doc/user/campaigns/index.md [hunk 1]
```diff
@@ -71,11 +69,11 @@ See this video for a demonstration of lifecycle of a campaign:
...
 
-1. With the `src` CLI the user **generates a set of patches** by running `gofmt` over every repository that has a `go.mod` file, leveraging Sourcegraphs search capabilities.
+1. The `src` CLI **generates a set of patches** by running `gofmt` over every repository that has a `go.mod` file, leveraging Sourcegraphs search capabilities.
 
-    This is called **executing an _action_** (an _action_ is a series of commands and Docker containers to run in each repository) and yields **set of patches**, one for each repository, which you can inspect either in the CLI or in the Sourcegraph UI.
+    This is called **executing an _action_** (an _action_ is a series of commands and Docker containers to run in each repository). It yields a **set of patches**, one for each repository, which you can inspect either in the CLI or in the Sourcegraph UI.
 1. The patches are then used to **create a draft campaign**.
-1. At this point, since it's a draft camapaign, no changesets (_pull requests_ in the case of GitHub here) have been created on the code host.
+1. At this point, since it's a draft camapaign, no changesets (or _pull requests_ in the case of GitHub) have been created on the code host.
```
Comment (LawnGnome) [inline doc/user/campaigns/index.md:76]: The "pull request" callout is also covered further up the same page: do we need to repeat it here? (Or, alternatively, should we consider creating a glossary?)

File: doc/user/campaigns/index.md [hunk 2]
```diff
@@ -87,4 +85,4 @@ See this video for a demonstration of lifecycle of a campaign:
 
 ## Limitations
 
-Campaigns currently only support **GitHub** and **Bitbucket Server** repositories. If you're interested in using campaigns on other code hosts, [let us know](https://about.sourcegraph.com/contact).
+Campaigns currently support **GitHub**, **GitLab** and **Bitbucket Server** repositories. If you're interested in using campaigns on other code hosts, [let us know](https://about.sourcegraph.com/contact).
```
Comment (LawnGnome) [inline doc/user/campaigns/index.md:88]: Good catch. Thanks!

File: doc/user/campaigns/index.md (hunk 1 above)
Comment (kghopson) [reply doc/user/campaigns/index.md:76]: Y'know, I like this glossary suggestion. I'll add it to my to-dos, but yeah this is probably redundant.

File: doc/user/campaigns/index.md (hunk 2 above)
Comment (kghopson) [reply doc/user/campaigns/index.md:88]: Ha, thank Laureen actually

File: doc/user/campaigns/index.md [hunk 3]
```diff
@@ -71,11 +69,11 @@ See this video for a demonstration of lifecycle of a campaign:
   </figure>
 </div>
 
-1. With the `src` CLI the user **generates a set of patches** by running `gofmt` over every repository that has a `go.mod` file, leveraging Sourcegraphs search capabilities.
+1. The `src` CLI **generates a set of patches** by running `gofmt` over every repository that has a `go.mod` file, leveraging Sourcegraphs search capabilities.
```
Comment (LaureenH) [inline doc/user/campaigns/index.md:72]: Sourcegraph's

Comment (kghopson) [reply doc/user/campaigns/index.md]: Fixed. Thanks for catching it!

**Pull request merged**